packages/core/src/tasks/index.ts

```diff
@@ -76,28 +77,28 @@ import {
 export const chugsplashRegisterAbstractTask = async (
   provider: ethers.providers.JsonRpcProvider,
   signer: ethers.Signer,
-  parsedConfig: ParsedChugSplashConfig,
+  config: UserChugSplashConfig | ParsedChugSplashConfig,
```

Comment:
It actually looks like we can replace this with `userConfig: UserChugSplashConfig` because this type is a subset of the `ParsedChugSplashConfig` type